### ISSUE TITLE
updateinfo issues

### ISSUE THREAD

**Issue by glum23** [rpm-software-management/dnf5#919]
According to https://github.com/rpm-software-management/dnf5/issues/155, work on `updateinfo` command has been finished. However:
- Documentation is insufficient. 
In https://dnf5.readthedocs.io/en/latest/changes.html, `updateinfo` is mentioned only once and the mention of it is incorrect (see the next item). It definitely should be listed among changed commands as it default name has been changed to `advisory` while many aliases (including `upif`) are not available now.
`dnf5 updateinfo --help` does not provide any options while several options are listed in the man page (vice versa problem is already tracked for all commands in https://github.com/rpm-software-management/dnf5/issues/644 )
- In the section "Changes on the command line" of https://dnf5.readthedocs.io/en/latest/changes.html, there's explicitly mentioned `dnf updateinfo info -> dnf updateinfo –info` as an example of the new approach in comparison with dnf4, however, the command is now implemented in the old way in dnf5.

**Comment by kontura** [comment]
Thank you for the report!
I have created a PR https://github.com/rpm-software-management/dnf5/pull/920 which documents the changes.

Regarding the `dnf5 updateinfo --help` not showing any options. I think that is expected, the options are for subcommands: `dnf5 updateinfo info --help`.

**Comment by glum23** [comment]
> Thank you for the report! I have created a PR #920 which documents the changes.
Thanks!

> Regarding the `dnf5 updateinfo --help` not showing any options. I think that is expected, the options are for subcommands: `dnf5 updateinfo info --help`.
I'd expect the output of  `dnf5 updateinfo --help` similar to the related man page (especially because all subcommands' options are the same), sth like:
~~~
Usage:
  dnf5 advisory <COMMAND> [GLOBAL OPTIONS] [OPTIONS] [ARGUMENTS]
Command:                 
  list                          List advisories
  info                          Print details about advisories
  summary                       Print summary of advisories
Options:                         
 <list of options>
Arguments:                                     
  advisory-spec                                List of patterns matched against advisory names.
~~~
From my POV, seeking the relevant options in two steps is unnecessarily protracted. However, the current solution is consistent.

**Comment by kontura** [comment]
They are the same for `updateinfo` but it doesn't have to be like that in general.
For example `dnf5 group list --help` vs `dnf5 group install --help`.

But I see your point about the man page. 
I agree it is something to consider.

**Comment by glum23** [comment]
new issue: https://github.com/rpm-software-management/dnf5/issues/921
